internal/resources/metal/vlan/models.go

```diff
@@ -56,12 +68,15 @@ type ResourceModel struct {
 	Description types.String `tfsdk:"description"`
 }
 
-func (m *ResourceModel) parse(vlan *packngo.VirtualNetwork) diag.Diagnostics {
+func (m *ResourceModel) parse(vlan *packngo.VirtualNetwork) (d diag.Diagnostics) {
 	m.ID = types.StringValue(vlan.ID)
-	m.Description = types.StringValue(vlan.Description)
 	m.Vxlan = types.Int64Value(int64(vlan.VXLAN))
 	m.Facility = types.StringValue("")
 
+	if vlan.Description != "" {
+		m.Description = types.StringValue(vlan.Description)
```

Comment:
Does this effectively prevent TF from recognizing an API cleared description?

Terraform initially set and continues to want: "foo"
API was updated to: ""
Terraform ignores the diff?

Reply:
VLAN doesn't have an update endpoint according to the spec & both SDKs (all attributes of VLAN resources are marked as requiring replacement, and have been for quite a while).  I added a test, though, to confirm that the description is removed; if we ever add support for updating VLAN attributes, that will help confirm that description continues to work as expected.